docs/source/ja/index.mdx

```diff
@@ -0,0 +1,392 @@
+<!--Copyright 2020 The HuggingFace Team. All rights reserved.
```

Comment:
```suggestion
<!--Copyright 2023 The HuggingFace Team. All rights reserved.
```

Reply:
Thank you for the review!
I fixed the year of the license from 2020 to 2023.